### ISSUE TITLE
Enforce Linked Issue on Pull Request

### ISSUE THREAD

**Issue by wba2hi** [eclipse-kuksa/kuksa-android-sdk#8]
**Is your feature request related to a problem? Please describe.**
As a developer I always want to have an existing github issue for a PullRequest, so that more information about the pull request is known. 

**Describe the solution you'd like**
When a pull request is created, a merge should not be possible without a linked github issue. compare it to a "failed build".
In optimal case, it is automatically recognized (without having to do a separate push or a new PR) if an issue is linked and unblocks the merge option, allowing a merge (after a positive review)
